src/engine/main.cpp

```diff
@@ -1041,6 +1041,7 @@ int main(int argc, char **argv)
     SDL_ShowCursor(SDL_FALSE);
     SDL_StopTextInput(); // workaround for spurious text-input events getting sent on first text input toggle?
 
+#ifdef NDEBUG
```

Comment:
Please document why we decided to disable the signals for debug builds in a comment.